src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -3416,10 +3420,10 @@ class MavlinkStreamAltitude : public MavlinkStream
 			struct home_position_s home;
 			updated |= _home_sub->update(&_home_time, &home);
 
-			if (_global_pos_time > 0 and _home_time > 0) {
+			if (_global_pos_time > 0 && _home_time > 0) {
 				msg.altitude_relative = global_alt - home.alt;
 
-			} else if (_local_pos_time > 0 and _home_time > 0) {
```

Comment:
<%
    if (argc > 1 and argv<:1:> not_eq NULL) <%
        std::cout << "Hello, " << argv<:1:> << '\n';
    %>
%>